scripts/run-doxygen.sh

```diff
@@ -49,14 +54,17 @@ cp ${MTOC_CONFIG_PATH}/latexextras.template ${MTOC_CONFIG_PATH}/latexextras.sty
 sed -i -e "s#_ConfDir_#${MTOC_CONFIG_PATH}#g" ${MTOC_CONFIG_PATH}/latexextras.sty
 
 doxygen "${DOXYFILE}"
-doxygen "${DOXYFILE}"
```

Comment:
I think there was some reason for running doxygen twice. Not sure anymore though...

Reply:
issue was fixed in #597b138